dev/com.ibm.ws.jaxrs.2.1_fat_extended/publish/servers/jaxrs21.fat.packageJsonBNoFeature/server.xml

```diff
@@ -6,13 +6,9 @@
 	
 	<include location="../fatTestPorts.xml"/>
 	
-	<javaPermission className="java.util.PropertyPermission" name="org.apache.johnzon.max-string-length" actions="read"/>
-	<javaPermission className="java.util.PropertyPermission" name="org.apache.johnzon.default-char-buffer" actions="read"/>
-	<javaPermission className="java.util.PropertyPermission" name="org.apache.johnzon.supports-comments" actions="read"/>
-	<javaPermission className="java.util.PropertyPermission" name="org.apache.johnzon.default-char-buffer-generator" actions="read"/>
-	<javaPermission className="java.util.PropertyPermission" name="johnzon.string-builder.strategy" actions="read"/>
-	<javaPermission className="java.util.PropertyPermission" name="johnzon.mapper.encoding" actions="read"/>
-	<javaPermission className="java.util.PropertyPermission" name="org.apache.johnzon.default-string-builder" actions="read"/>
+	<javaPermission className="java.util.PropertyPermission" name="*" actions="read"/>
```

Comment:
It seems like a valid requirement to be able to only grant permissions to read johnzon properties vs every property.  Can this be done with a org.apache.johnzon.* and johnzon.* or is that syntax not supported for PropertyPermission?